Add clarity to Pulumi import command (#3536)
pulumi/pulumi-hugo: themes/default/content/docs/using-pulumi/adopting-pulumi/import.md

### Pulumi Import Command

> When importing resources via the CLI, there is a specific import syntax definition required for each resource. To see this definition, navigate to the corresponding resource page in the Pulumi registry and take a look at the **Import** section. Using the [AWS S3 Bucket resource](https://www.pulumi.com/registry/packages/aws/api-docs/s3/bucket/#import) as an example, the below code demonstrates how to import an existing AWS S3 bucket named `company-infra-logs` and defines the resource name for your Pulumi program as `infra-logs`:

Reviewer: ```suggestion
When importing resources via the CLI, there is a specific import syntax definition required for each resource. To see this definition, navigate to the corresponding resource page in the Pulumi registry and take a look at the **Import** section. Using the [AWS S3 Bucket resource](https://www.pulumi.com/registry/packages/aws/api-docs/s3/bucket/#import) as an example, the following command imports an existing AWS S3 bucket named `company-infra-logs` and defines the resource name for the Pulumi program as `infra-logs`:
```

Reviewer: Dang! The auto-merge setting merged before my suggestion could be applied. I'll submit as a follow up.
